src/lib/merkle_mask/masking_merkle_tree.ml

```diff
@@ -613,6 +624,33 @@ module Make (Inputs : Inputs_intf.S) = struct
       self_find_or_batch_lookup self_find_location
         Base.location_of_account_batch
 
+    let unsafe_preload_accounts_from_parent t account_ids =
```

Comment:
@mrmr1993 A comment on when this could be used or should not be used would be helpful

Reply:
Addressed